pkg/monitoring/client/prometheus/prometheus.go

```diff
@@ -51,8 +60,9 @@ func init() {
 	prometheus.MustRegister(requestLatency)
 	prometheus.MustRegister(requestResult)
 	metrics.Register(metrics.RegisterOpts{
-		RequestLatency: &latencyAdapter{requestLatency},
-		RequestResult:  &resultAdapter{requestResult},
+		RequestLatency:     &latencyAdapter{requestLatency},
+		RateLimiterLatency: &latencyAdapter{rateLimiterLatency},
```

Comment:
I could be missing something, but I don't see where `RateLimiterLatency` gets populated with data.

Reply:
That is done via the latencyAdapter methods.

Reply:
I know but who uses them?  I don't see `Observe` or `Increment` called in the code base.

Reply:
Here it is: https://github.com/kubevirt/kubevirt/blob/main/vendor/k8s.io/client-go/rest/request.go#L598

Reply:
ah ok it comes from client-go, thanks